meta-mel/recipes-core/busybox/busybox_%.bbappend

```diff
@@ -8,4 +10,5 @@ SRC_URI_append_mel = "\
     file://fancy-head.cfg \
 	file://pidof.cfg \
 	file://top.cfg \
+    ${@bb.utils.contains('INCOMPATIBLE_LICENSE', 'GPLv3', bb.utils.contains('IMAGE_FEATURES', 'security-selinux', 'file://chcon.cfg', '', d), '', d)} \
```

Comment:
I'd suggest setting up a variable above on this basis and then adding that to SRC_URI. That would be more flexible.

Reply:
Done. Please have a look

Reply:
Just a note that checking image features outside the image is highly specific to how we do things in mel/sokol-flex, upstream would never accept it. Not an issue for this PR, this is fine, but just so people know.. from a yocto perspective image features can be set entirely in the image, it's just us that set it globally more often than not :)

Reply:
Thanks for sharing this. It is good to know the proper way to use it.